Assignment1/assignment2/lib/utilities.dart

```diff
@@ -0,0 +1,112 @@
+import 'package:assignment2/users.dart';
+import 'dart:collection';
+
+Map<int, User> currentSession = Map();
+
+void nameHandler(String name) {
```

Comment:
Can be renamed to 'nameValidator' as this doesn't handle the name rather validates it.

Reply:
done

Reply:
done